tools/dotnet-deb-tool/project.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc3-004308"
+      "version": "1.0.0"
     }
   },
```

Comment:
I believe this is also running on stage0 which is why it wasn't star versioned before as well.